src/main/java/org/jabref/gui/icon/JabRefIcon.java

```diff
@@ -3,16 +3,21 @@
 import javafx.scene.Node;
 import javafx.scene.paint.Color;
 
-import de.jensd.fx.glyphs.GlyphIcons;
+import org.kordamp.ikonli.Ikon;
 
-public interface JabRefIcon extends GlyphIcons {
+public interface JabRefIcon {
 
     Node getGraphicNode();
 
-    @Override
     String name();
 
     JabRefIcon withColor(Color color);
 
     JabRefIcon disabled();
+
+    String fontFamily();
```

Comment:
Is the fontFamily and unicode still necessary?  In either way, I would add a "get" to the name